olm-catalog/serverless-operator/metadata/annotations.yaml

```diff
@@ -1,6 +1,6 @@
 annotations:
     operators.operatorframework.io.bundle.channel.default.v1: "4.3"
```

Comment:
I guess we'll have to update this to something sensible for this version? :thinking: